zstacklib/zstacklib/utils/lvm.py

```
@@ -2746,10 +2751,54 @@ def check_lvmlockd_log():
 def report_config_changed():
     shell.run("touch %s" % LVM_CONFIG_CHANGED_FILE)
 
+need_locking_cmds = ["lvchange", "lvcreate", "lvrename", "lvresize", "lvextend", "lvremove"]
+nolocking_cmds = ["lvs", "pvs", "vgs"]
 
 def subcmd(cmd, timeout=lvm_cmd_timeout_with_locking):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)